### PR TITLE
Support for DNACs, CNR and ancient firmware

### PR DESCRIPTION
**Pull request by ecoen66**
Added support for DNA Center and CNR appliances.  Added allowance for ancient IMC firmware.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -17,14 +17,13 @@
     history = history_file.read()
 
 setup(
-    name='imcsdk',
+    name='imcsdk_ecoen66',
```
Comment (vvb) [inline setup.py]: Changes in setup.py need to be reverted for this to be merged.

**Review by vvb — CHANGES_REQUESTED** [review]
please revert changes done in setup.py

**Review by ecoen66 — COMMENTED** [review]
Changes removed from setup.py

**Review by ecoen66 — COMMENTED** [review]
Added DN3 DNAC

**Review by vvb — APPROVED** [review]
LGTM

**Pull request merged**